app/src/main/res/values/strings.xml

```diff
@@ -499,6 +497,8 @@
   <string name="help_activity_back_arrow_description">Navigate back to %s</string>
   <string name="help_activity_third_party_dependencies_list">third-party dependencies list</string>
   <string name="help_activity_copyright_licenses_list">copyright licenses list</string>
+  <string name="faq_activity_title">FAQs (Frequently Asked Questions)</string>
```

Comment:
Hmm, this also belongs to the group of strings for the FAQ list activity.

I'm actually thinking that we might need more of a strategy for organizing these strings. #3384 is specific to HelpActivity, but we really should be thinking about these generically so that it's really obvious where a string should go.

@rt4914 you've had more context into strings being added here than I have. Do we have any sort of strong organization rules for this file that would apply here and, if not, do you have any initial ideas on what those should be?

Reply:
Hey @BenHenning . I was thinking, maybe we can combine FAQ and Help Activity strings and create a hierarchy

//Help and FAQ Activity

                //Help Activity
                ...
                //FAQ Activity

Reply:
@BenHenning I don't have any strong rules. But I thinking the structure can be something like this:
// All common strings on top or any string which gets used at multiple places.
// All activities in alphabetical order like:
```
<!-- HelpActivity -->
<string name="help_title" >Help</string>
<string >...</string>

<!-- HomeActivity -->
<string name="home_title" >Home</string>
<string >...</string>
```
WDYT?

Reply:
@aadityaguptaa once concern I have with that is that activities aren't actually hierarchical. They can theoretically be opened from anywhere within the app (& some like the options & topic activities are opened from different places). Activities form more of a graph than a tree.

@rt4914 that could work, though maybe we shouldn't have common strings at all--this is actually a problem with the current strings. Some are too generic and lack the context of where they're used. If we are keeping all strings specific, then we can also have a common naming style for them, e.g. home_activity_title, home_activity_greeting_header, etc. Fragments could probably be bundled under their corresponding activities since we generally don't share fragments across activities even though they technically could be.

Thoughts?

Reply:
Does make sense, especially keeping in mind the localisation factor.

Reply:
In that case, I think we either need to change this PR to be reorganizing all strings to be based on their corresponding activities (which ideally would include fixes to the string names themselves), otherwise this one change doesn't really help that much with consistency since most of the other strings are also out of order.

@rt4914 do you have a recommendation for how @aadityaguptaa should proceed here?

Reply:
To clarify: I'm not sure just changing two strings makes sense without figuring out the full strategy for organizing strings. I think that's a prerequisite before we can actually put the strings in their final position (and that may require renaming strings). I don't mind if we do this in multiple PRs, though, given how many strings there are.